concrete/src/Command/Task/Runner/Context/ConsoleContext.php

```diff
@@ -1,14 +1,46 @@
 <?php
+
 namespace Concrete\Core\Command\Task\Runner\Context;
 
+use Concrete\Core\Application\Application;
+use Concrete\Core\Command\Task\Messenger\ConsoleTaskMessengerEventSubscriber;
+use Concrete\Core\Command\Task\Output\OutputInterface;
 use Concrete\Core\Command\Task\Stamp\OutputStamp;
 use Concrete\Core\Messenger\Stamp\SkipSendersStamp;
+use Concrete\Core\Messenger\Transport\TransportInterface;
+use PortlandLabs\Concrete5\MigrationTool\Messenger\PublisherMessengerEventSubscriber;
```

Comment:
@aembler 

In my opinion, we shouldn't use a package's class in the core.

Reply:
Nice catch...resolved in https://github.com/concretecms/concretecms/pull/12392